### PR TITLE
 Wrong design for project name error

### PR DESCRIPTION
**Pull request by sajdakabir**
Close: #4652 

## PR Checklist

- [X] Tests for the changes have been added
- [X] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -426,6 +426,7 @@
       "version": "2.2.0",
       "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.0.tgz",
       "integrity": "sha512-qRmjj8nj9qmLTQXXmaR1cck3UXSRMPrbsLJAasZpF+t3riI71BXed5ebIOYwQntykeZuhjsdweEc9BxH5Jc26w==",
+      "dev": true,
```
Comment (overbit) [inline package-lock.json:429]: Also these changes to the package-lock are unrelated.

**Review by overbit — CHANGES_REQUESTED** [review]

**Comment by sajdakabir** [comment]
@overbit can you please check it once again?

**Review by EugeneTseitlin — CHANGES_REQUESTED** [review]
Please, don't delete package-lock.json

**Comment by sajdakabir** [comment]
Hey, @EugeneTseitlin sorry for this :(. Is it okay now?

**Comment by yuval-hazaz** [comment]
@sajdakabir thanks for you support.
I just saw that another PR was accepted to resolve this issue (https://github.com/amplication/amplication/pull/5091) 
but, since you were assigned to it, we will accept your PR once it is complete. 
Please also have a look at the solution in https://github.com/amplication/amplication/pull/5091 - and see if you want to use anything from there.

**Comment by EugeneTseitlin** [comment]
> Hey, @EugeneTseitlin sorry for this :(. Is it okay now?

Still not. Please, revert deleting package-lock.json. You can check the changes in "Files changed" tab. package-lock.json should not be mentioned there.

**Comment by sajdakabir** [comment]
@EugeneTseitlin can you check now?

File: .github/workflows/release.template.yml [hunk 2]
```diff
@@ -115,7 +115,8 @@ jobs:
         env:
           INPUT_TAGS: ${{ steps.meta.outputs.tags }}
           INPUT_PUSH: true
-
+          INPUT_BUILD_ARGS: GIT_REF=${{ github.ref }}
```
Comment (EugeneTseitlin) [inline .github/workflows/release.template.yml:118]: This change is not related to the PR.

**Comment by EugeneTseitlin** [comment]
@sajdakabir Thank you for the contribution! 
After a deeper look into the issue and consulting with the team, we've came to a conclusion that an actual scope of the issue is wider than we thought and we've decided to review every usage of TextField component and make sure that it's aligned with the design system. We're planning to solve this issue by the team effort, so it's not available for community. We really appreciate your effort, but unfortunately the PR can't be accepted, because our plan has changed. 
We're welcome to PRs from a community, so feel free to request an assignment for another issue with "open for community" tag.
Thanks for being with us!

**Pull request closed without merging**